app/forms/declaration/destinationCountries/DestinationCountries.scala

```diff
@@ -16,12 +16,63 @@
 
 package forms.declaration.destinationCountries
 
-import play.api.data.Form
+import play.api.data.Forms.{default, seq, text}
+import play.api.data.{Form, Forms, Mapping}
+import play.api.libs.json.{Json, OFormat}
+import services.Countries.allCountries
+
+case class DestinationCountries(
```

Comment:
Just to highlight I've combined the Supplementary and Standard form. 
Reasons for doing this are:
- Countries Standard wasnt shown on the summary page
- Fixing the above meant pulling CountriesStandard out of the cache which failed with Deserializer errors because that and CountriesSupplementary have the same  formID
- Theres only one field extra in standard so why not use the same class with different forms

Reply:
I'd be more tempted to fix the existing implementation as part of another ticket. Sounds like it needs fixing but I dont think combining these is the correct approach.